docs/workflows/create_sync_publish.rst

```diff
@@ -38,6 +38,11 @@ RPM Repositories support several additional options.
     See :ref:`metadata_signing`.
 - retain_package_versions:
     The maximum number of versions of each package to keep; as new versions of packages are added by upload, sync, or copy, older versions of the same packages are automatically removed. A value of 0 means "unlimited".
+- autopublish:
+   If set to True, will automatically create publications for new repository versions, and update any distributions pointing to this repository. When using this option, after repository has been created, point the distribution to it, namely set the `repository` field.
+
+Pulpcore provides `retain_repo_versions` setting that can also be set. For example, if set to 1, it will make sure it will keep at minimum 1 repo version and rest will be automatically deleted, together with its associated publication.
```

Comment:
I'm not sure if the fact that it is provided by pulpcore matters to a user.  I can see arguments both ways, on one hand it's irrelevant in the immediate sense to someone reading the RPM documentation, on the other hand if they use other content types the reminder could be useful?

Reply:
I decided to mention pulpcore to create a thought in the user's head " what if there is more stuff I can go and checkout"
We do not want to duplicate all pulpcore options in the plugin, but still we somehow need to raise the awareness of their existence.
I created these docs as follow up of conversation I had with a user on the ticket. He seemed to be pretty well informed of rpm related options but not pulpcore, so my assumption is he's mostly replying/reading rpm docs.
Also maybe this assumption is not true but our pulpdore cdocs are big and overwhelming and not very nicely structured so there is anyway a high chance of overlooking something useful.

Reply:
This PR did not mean to solve our docs issue but more was to be inteded as emergency duck-taping :)